<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SchedulerDialog</class>
 <widget class="QDialog" name="SchedulerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scheduled Run Configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QRadioButton" name="noScheduleButton">
     <property name="text">
      <string>No scheduled execution</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="scheduleButton">
     <property name="text">
      <string>Scheduled execution</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="configurationGroupBox">
     <property name="title">
      <string>Schedule configuration</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QRadioButton" name="dailyButton">
            <property name="text">
             <string>Daily</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="weeklyButton">
            <property name="text">
             <string>Weekly</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="monthlyButton">
            <property name="text">
             <string>Monthly</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>at</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="scheduleTimeEdit"/>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QStackedWidget" name="configurationStackedWidget">
            <property name="currentIndex">
             <number>2</number>
            </property>
            <widget class="QWidget" name="dailyPage">
             <layout class="QHBoxLayout" name="horizontalLayout_5"/>
            </widget>
            <widget class="QWidget" name="weeklyPage">
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <layout class="QGridLayout" name="gridLayout">
                <item row="0" column="0">
                 <widget class="QCheckBox" name="sundayBox">
                  <property name="text">
                   <string>Sunday</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QCheckBox" name="wednesdayBox">
                  <property name="text">
                   <string>Wednesday</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QCheckBox" name="mondayBox">
                  <property name="text">
                   <string>Monday</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QCheckBox" name="thursdayBox">
                  <property name="text">
                   <string>Thursday</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="2">
                 <widget class="QCheckBox" name="saturdayBox">
                  <property name="text">
                   <string>Saturday</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QCheckBox" name="tuesdayBox">
                  <property name="text">
                   <string>Tuesday</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QCheckBox" name="fridayBox">
                  <property name="text">
                   <string>Friday</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="montlyPage">
             <layout class="QVBoxLayout" name="_2"/>
            </widget>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="PathSelectionControl" name="fileSelectionDialog" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PathSelectionControl</class>
   <extends>QWidget</extends>
   <header location="global">pathselectioncontrol.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SchedulerDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>322</x>
     <y>498</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
